redaxo/src/core/lib/console/application.php

```diff
@@ -54,7 +54,16 @@ private function loadPackages(rex_console_command $command)
         // Some packages requires a working db connection in their boot.php
         // in this case if no connection is available, no commands can be used
         // but this command should be always usable
-        if ('db:set-connection' === $command->getName()) {
+        if ($command instanceof rex_command_disable_packages) {
+            return;
+        }
+
+        // Loads only setup packages
+        // This is useful for any kind of pre-setup commands
+        if ($command instanceof rex_command_only_setup_packages) {
+            foreach (rex_package::getSetupPackages() as $package) {
+                $package->enlist();
```

Comment:
warum `enlist`'ten wir in diesem case nur.. in den anderen cases `boot`'ten wir aber`?

Reply:
stimmt.